### PR TITLE
[Fixes #6984] ISO category translation not shown (german)

### LINKED ISSUE #6984: ISO category translation not shown (german)
## Expected Behavior
The ISO category (oceans, imagery base maps ...) are translated to their respective counterpart.

## Actual Behavior
The English string will be used all over the UI. Example:

grafik

## Steps to Reproduce the Problem

  1. switch language to german
  2. take a look at the iso strings

## Specifications

  - GeoNode version: master
  - Installation method (manual, GeoNode Docker, SPCGeoNode Docker): all

  - Additional details:

This [commit](https://github.com/GeoNode/geonode/blame/master/geonode/locale/de/LC_MESSAGES/django.po#L8754) did comment out many translation strings in the german .po file. That is why those are not used anymore. I have uncommented those needed for the translation again and will provide a PR for the german translation, soon.

### PR DESCRIPTION
**Pull request by gannebamm**
uncommented msgids and some minor fixes

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in the documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for contrib modules):
- [x] There is a ticket in https://github.com/GeoNode/geonode/issues describing the issue/improvement/feature (a notable exemption is, changes not visible to end-users)
- [x] The issue connected to the PR must have Labels and Milestone assigned
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[Fixes #<issue_number>] Title of the Issue"
- [ ] New unit tests have been added covering the changes, unless there is an explanation on why the tests are not necessary/implemented
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the QA checks: flake8 geonode
- [ ] Commits changing the **settings**, **UI**, **existing user workflows**, or adding **new functionality**, need to include documentation updates

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

### CODE REVIEW

File: geonode/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -790,7 +788,7 @@ msgstr "Offline"
 #: geonode/base/templates/base/_resourcebase_snippet.html:60
 #: geonode/layers/templates/layers/layer_detail.html:46
 msgid "Layer not ready yet. Still finalizing layer ingestion..."
-msgstr ""
+msgstr "Ebene noch nicht fertig. Ebene wird noch eingeladen..."
```
Comment (t-book) [inline geonode/locale/de/LC_MESSAGES/django.po:791]: Das "eingeladen" hört sich etwas technisch für mich an. Vielleicht einfach:
"Die Ebene wird verarbeitet ..." ?

Comment (gannebamm) [reply geonode/locale/de/LC_MESSAGES/django.po:791]: zustimmung

Comment (t-book) [reply geonode/locale/de/LC_MESSAGES/django.po]: ja sehr gut!

**Comment by t-book** [comment]
Danke Florian 👍

**Pull request merged**